src/invidious/frontend/comments_youtube.cr

```diff
@@ -89,6 +89,20 @@ module Invidious::Frontend::Comments
               </div>
               END_HTML
             end
+          when "playlist"
+            if attachment["error"]?
+              html << <<-END_HTML
+              <div class="pure-g video-iframe-wrapper">
+                <p>#{attachment["error"]}</p>
+              </div>
+              END_HTML
+            else
+              html << <<-END_HTML
+              <div class="pure-g video-iframe-wrapper">
+                <iframe class="video-iframe" src='/embed/#{attachment.dig?("videos", 0, "videoId")}?autoplay=0&list=#{attachment["playlistId"]}'></iframe>
+              </div>
```

Comment:
On Youtube it's showing the first video too?

Reply:
YouTube renders video and playlist attachments in the same way as in the `video` and `playlist` tab (so it wouldn't use embeds for either). I can remove the playlist rendering part of this code if preferred. I think in the future it'd be beneficial to be able to use item.ecr for the video and playlist attachments (Comments would need to be refactored even more if that were to happen)

Reply:
> I can remove the playlist rendering part of this code and only keep the quiz parsing if that's preferred.

Yes, please! I think a lot needs to be refactored in comments/community before we can cleanly adding new UI elements.

> I think in the future it'd be beneficial to be able to use item.ecr for the video and playlist attachments instead of using the embeds

Hum, I don't know... But that's a discussion for a separate issue ^^